tests/api/processes_test.go

```diff
@@ -3,9 +3,37 @@ package api_test
 import (
 	"testing"
 
-	"github.com/bgentry/heroku-go"
+	heroku "github.com/bgentry/heroku-go"
```

Comment:
For some reason my goimports plugin keeps deleting heroko-go otherwise. I'll fix...